manifest.yml

```diff
@@ -5,6 +5,7 @@ applications:
   - java_buildpack
   health-check-type: http
   health-check-http-endpoint: /actuator/health
+  health-check-timeout: 180
```

Comment:
Will this be enough time on staging where we will be updating about half a million records?

Reply:
This is the maximum we can configure without asking the PaaS team to extend it. The update has already happened as part of the failed build, so the next build won't take much time at all.